CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_greenhouse_source v0.7.0
+[PR #29](https://github.com/fivetran/dbt_greenhouse_source/pull/29) includes the following updates:
+
+## Bug Fixes
+- Updated the logic for `stg_greenhouse__tag` and `stg_greenhouse__user` to account for the presence of the singularly or plurally-named titular source tables, tag(s) and user(s).
+- This is a breaking change for customers with the plurally-named tables, as they have not been able to run the models previously.
```

Comment:
- I think it would be easier for a customer if the source table versions are explicitly spelled out instead of being descriptive. Something along the lines of connectors after ____ date are TAGS and before it's TAG. 
- Also I would link to any relevant release notes.